index.html

```diff
@@ -6,17 +6,19 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
+  <script src="script.js" defer></script>
   <script src="https://kit.fontawesome.com/89d87f2112.js" crossorigin="anonymous"></script>
   <title>Portafolio</title>
 </head>
 
 <body>
     <header class="header">
+      <div class="crossed"></div>
       <div class="tallbar"></div>
       <div class="header-sub">
         <a href="" class="aa"><h2 class="logo">My logo</h2></a>
-        <nav class="bb">
-          <ul class="nav-elements">
+        <nav>
+          <ul class="nav-elements bb">
             <li><a href="#work">portifolio</a></li>
```

Comment:
- [ ] Kindly, make sure when the user clicks on any of the mobile menu options, the correct part of the page is displayed, in order to, follow the project's requirements. Currently, when I click on the **portfolio** link the path doesn't change. 
![image](https://github.com/tatouroberto/portfolio-project/assets/86069740/7879e282-320c-454e-8771-a54035737244)